examples/benchmarks/lstm_tagger.py

```diff
@@ -59,7 +59,7 @@ def build_tagger(
     embed: Model[Array2d, Array2d],
     encode: Model[Padded, Padded],
     predict: Model[Array2d, Array2d],
-) -> Model[List[Array2d], Padded]:
+) -> Model[List2d, List2d]:
```

Comment:
`encode` outputs `Padded`, shouldn't the output be `Padded`?

Reply:
@richardpaulhudson: I would revert the changes to this file and log an Ora ticket that we need to look into this one in more detail as a follow-up (I would do the same for the CI settings in `azure-pipelines.yml`)

Reply:
I have created the Ora ticket and will revert this file in this PR.